src/test/java/systemtests/contacts/ContactsSystemTest.java

```diff
@@ -40,10 +40,11 @@ protected void selectPerson(Index index) {
     }
 
     /**
-     * Deletes all persons in the address book.
+     * Deletes everything in the address book.
      */
-    protected void deleteAllPersons() {
-        executeCommand(MODULE_WORD + " " + ClearCommand.COMMAND_WORD);
+    protected void clear() {
+        executeCommand(ClearCommand.COMMAND_WORD);
         assertEquals(0, getModel().getAddressBook().getPersonList().size());
+        assertEquals(0, getModel().getAddressBook().getTaskList().size());
```

Comment:
Don't think this belongs here after your changes. Maybe move it up to `AppSystemTest`? This file is supposed to contain commands specific to contacts.

Might need to `@Override` in `ClearCommandSystemTest` after that.

Reply:
This is used in a lot of the contacts systems tests. Unless we're removing all those parts after `clear()` because they assume that all persons are deleted and we have no command specifically just to delete all `Person`s[](url)

Reply:
Right, I misunderstood what you meant

Reply:
Done